### ISSUE TITLE
TFCamembertModel

### ISSUE THREAD

**Issue by kintzinj** [huggingface/transformers#2683]
Hi, 

I wanted to use the tensorflow version of Camembert : TFCamembertModel, but the implememtation is not available with the v2.3.0 version : https://huggingface.co/transformers/v2.3.0/model_doc/camembert.html. 

But TFCamembertModel  seems to be available with another version of transformers : https://huggingface.co/transformers/model_doc/camembert.html.

Is this a new or old version of the library ?

Any way, have you already succeeded importing TFCamembertModel ?

Thanks a lot !

**Comment by LysandreJik** [comment]
Hi, the CamemBERT model for tensorflow was merged yesterday, and is therefore available from the master branch right now. You can install it with 

```py
pip install git+https://github.com/huggingface/transformers
```

It will be in the next transformers release (2.4.0 or 2.3.1), which should be released later today.

**Comment by kintzinj** [comment]
Awesome, thank you !

**Comment by kintzinj** [comment]
OSError: Model name 'camembert-base2' was not found in tokenizers model name list (camembert-base).

OSError: Model name 'camembert-base2' was not found in model name list.

**Comment by kintzinj** [comment]
Hi again Lysandre,

I have checked the new version of Transformers and an error occurred when I tried to load TFCamembertModel:

`TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType`

I think I found what is missing. The pretrained models are not available, I mean the list is empty when I try to load a pretrained model.

`OSError: Model name 'test_list' was not found in model name list. We assumed 'https://s3.amazonaws.com/models.huggingface.co/bert/test_list/config.json' was a path, a model identifier, or url to a configuration file named config.json or a directory containing such a file but couldn't find any such file at this path or url.`

So I checked the pretrained model archives avaible in "modeling_tf_camembert.py" and indeed, the dictionnary is empty :

`TF_CAMEMBERT_PRETRAINED_MODEL_ARCHIVE_MAP = {}`

Is it on purpose ?

Thanks

**Comment by LysandreJik** [comment]
Hi, indeed there are not checkpoints in the archive map. The contributor (@jplu) that contributed the tensorflow architecture has uploaded checkpoints that you can use: https://huggingface.co/jplu/tf-camembert-base.

We're currently working on our website so that it better reflects the following:
- The CamemBERT model has official weights that are usable in PyTorch, but do not currently have any TensorFlow equivalent
- There are community models (cf. jplu/tf-camembert-base) which can be used instead.

We're trying to make sure that contributing models is easy and that weights are easily identifiable for users. This is still a work in progress.

cc @julien-c @joshchagani 

TLDR: in transformers v2.4.0, the following should work:

```py
from transformers import TFCamembertModel
model = TFCamembertModel.from_pretrained("jplu/tf-camembert-base")
```
